### PR TITLE
feat: explore -part1- fetch hotscenes

### PR DESCRIPTION
**Pull request by pbosio**
*fetch scenes across realms with people in it
*added hidden chat command to show the first 5 scenes with people in it (helpful for debugging porpoises and meeting new people :P)

### CODE REVIEW

File: kernel/packages/shared/social/hotScenes.ts [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    baseCoord: baseCoord.split(',').map((str) => parseInt(str, 10)) as ParcelArray,
+    scene: sceneJsonData,
+    usersCount: 1
+  }
+}
+
+async function getCrowdedScenesFromCandidate(candidate: Candidate): Promise<CandidateCrowdedScene[]> {
+  let scenes: Record<string, CandidateCrowdedScene> = {}
+
+  const tiles = candidate.layer.usersParcels?.map((value) => `${value[0] | 0},${value[1] | 0}`) ?? []
```
Comment (moliva) [inline kernel/packages/shared/social/hotScenes.ts]: If `usersParcels` is populated, its values ought to be properly initialized. If still, we want to catch any issue with them I would filter out the parcel before taking it into account, as we don't actually know the proper coordinate.

File: kernel/packages/shared/social/hotScenes.ts [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+  for (const tile of tiles) {
+    let id = (await fetchSceneIds([tile]))[0]
+    let land = null
+
+    if (id) {
+      land = (await fetchSceneJson([id]))[0]
```
Comment (moliva) [inline kernel/packages/shared/social/hotScenes.ts]: Might prove better to fetch both ids and data for the whole set of tiles before iterating though the collection.

**Review by moliva — CHANGES_REQUESTED** [review]

**Review by moliva — APPROVED** [review]
Great work! 😄

**Review by pablitar — APPROVED** [review]
LGTM!

**Pull request merged**